menu/menu.yaml

```diff
@@ -577,6 +577,10 @@
   - Url: transports
     Title: General
     Articles:
+    - Url: transports/selecting
```

Comment:
is it only me that thinks that `transports/selecting` is a kind of horrible URL?

Reply:
It's not the best, but can we think of something better?

Reply:
Leaving as-is for now.